src/v2/controllers/FormController.js

```diff
@@ -232,6 +236,17 @@ export default Controller.extend({
       });
   },
 
+  isBiometricsErrorFlow(error) {
+    if (!error?.context?.messages) {
+      return false;
+    }
+
+    const messagesObjs = error.context.messages.value;
+    const biometricsErrorKeys = [OV_UV_ENABLE_BIOMETRICS_FASTPASS_DESKTOP, OV_UV_ENABLE_BIOMETRICS_FASTPASS_MOBILE];
+    return messagesObjs && Array.isArray(messagesObjs)
+    && messagesObjs.some(messagesObj => _.contains(biometricsErrorKeys, messagesObj.i18n?.key));
```

Comment:
It does not seem reliable to depend on an i18n key from the response. Is there other signals like error type that you can check?

Reply:
> It does not seem reliable to depend on an i18n key from the response. Is there other signals like error type that you can check?

@jhe-okta here is the sample API response. I can add a class check of `ERROR` in this method.

Reply:
@yannongli-okta can you add the link?

Reply:
@jhe-okta sorry here it is https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/data/idp/idx/error-okta-verify-uv-fastpass-verify-enable-biometrics-desktop.json#L10

Reply:
Good catch @jhe-okta .. we should avoid view specific logic from FormController..

Reply:
Instead use `showCustomFormErrorCallout` attribute 

search in the repo and see how it is being used.. 

for eg take a look at `ChallengeOktaVerifyFastPassView.js`

Reply:
Ignore earlier comments.. Is that error not coming as IDX response ?